stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -537,11 +536,7 @@ internal fun MessageFooter(
                 color = ChatTheme.colors.textLowEmphasis
             )
 
-            Text(
-                SimpleDateFormat.getTimeInstance().format(message.createdAt ?: Date()),
-                style = ChatTheme.typography.footnote,
-                color = ChatTheme.colors.textLowEmphasis
-            )
+            Timestamp(date = message.updatedAt ?: message.createdAt)
```

Comment:
[general] Do we not name smaller components like this example to be named `TimestampText` : cause i feel the current name could be cross referenced as a `model` layer

Reply:
So I wouldn't name this `TimestampText` because if we decide to add more components to it - e.g. an icon for some case, or something more, then it's no longer just a text.

I think if we're creating some `model` layer class, we'd name it a bit more specifically - like `MessageTimestamp`. or `TimestampItem` or `TimestampData` or something similar.

Does this make sense? Do you have any other idea of what we could name it that's not explicitly exposing a `Text` in it?

Reply:
For some odd reason i think `component` Suffix could be good alternative to our Compose widgets per say but i got the idea

Reply:
this can be discussed later , resolving it for now @filbabic